ruby_exercises/mythical-creatures/lib/unicorn.rb

```diff
@@ -0,0 +1,18 @@
+class Unicorn
+    attr_reader :name
+
+    attr_accessor :color
```

Comment:
I think this can be put as an attr_reader. You'll use attr_accessor if you need to modify the value outside of the object (like trying to re-set the value of color on an instance of Unicorn).

```
my_unicorn = Unicorn.new("Jasmine")
my_unicorn.color = "blue" <- can't do this without have `color` in the `attr_accessor`
```